### PR TITLE
Added a new History section to the README, pointing to a separate cha…

### PR DESCRIPTION
**Pull request by alaric-dfe**
…ngelog and ADRs

Implements https://github.com/DFE-Digital/dfe-reference-data/issues/23

And, as a demo of the Architectural Decision Record process, includes a decision made in Slack earlier today.

### CODE REVIEW

**Review by frankieroberto — COMMENTED** [review]
couple of minor copy suggestions, removing ~~extraneus~~ words.

File: docs/decisions/0001-cycle-dependent-lists.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Handle lists that change between cycles using hash fields
+
+## Context and Problem Statement
```
Comment (edwardhorsford) [inline docs/decisions/0001-cycle-dependent-lists.md:3]:
```suggestion
## Context and problem statement
```

File: docs/decisions/0001-cycle-dependent-lists.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+1. Code that needs to present the user with a choice needs to be able to easily show the choices available for the cycle in question.
+2. Code that processes records need to find the appropriate properties for that record in a given cycle.
+3. Code that performs analytics or other operations on records from a mixture of cycles needs to be able to easily work with options that have NOT changed between cyclesA.
+   1. This may be complicated somewhat when list entries "split" or "merge"; handling these complications is context-specific so outside the scope of this decision.
+     1. An option for "Science" in cycle 1 might be split into separate "Physics", "Chemistry", "Biology", "Geology", etc in cycle 2, so a query for all "Science" records would need to know to include all of those options.
+     2. Meanwhile, a query for all "Physics" records would have to decide how to handle records using the "Science" code, which did not distinguish Physics from other sciences.
+     3. And that's just a simple case. The meaning of an "Other" option might change purely because a more specific option was added or removed elsewhere, for instance.
+
+## Considered Options
```
Comment (edwardhorsford) [inline docs/decisions/0001-cycle-dependent-lists.md:22]:
```suggestion
## Considered options
```

File: docs/decisions/0001-cycle-dependent-lists.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+* Store a separate reference list per cycle, with any records that don't change between cycles being duplicated.
+* Store a single reference list, but use fields that contain a hash from cycle ID to appropriate cycle-specific attributes of the record - including, if ncessary, a boolean to indicate its availability in that cycle (where that isn't implicit from the presence of other cycle-specific attributes).
+
+## Decision Outcome
```
Comment (edwardhorsford) [inline docs/decisions/0001-cycle-dependent-lists.md:27]:
```suggestion
## Decision outcome
```

File: docs/decisions/0001-cycle-dependent-lists.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+| ID | Name | Code By Cycle |
+|:-----|:-----|:-----|
+| 1 | Science | {1: SCI} |
+| 2 | English | {1: ENG, 2: ENG} |
+| 3 | Other | { 1: OTH, 2: OTH} |
+| 4 | Physics | {2: PHY} |
+| 5 | Biology | {2: BIO} |
+| 6 | Chemistry | {2: CHE} |
+
+### Additional Guidance
```
Comment (edwardhorsford) [inline docs/decisions/0001-cycle-dependent-lists.md:68]:
```suggestion
### Additional guidance
```

File: docs/decisions/0001-cycle-dependent-lists.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Handle lists that change between cycles using hash fields
+
+## Context and Problem Statement
+
+Some reference data lists change between cycles.
```
Comment (edwardhorsford) [inline docs/decisions/0001-cycle-dependent-lists.md:5]: Might want to define what a cycle is? or else refer to academic years?

**Pull request merged**